### PR TITLE
feat : added styles and structure for auth pages

### PR DESCRIPTION
**Pull request by aatishpandey**
Auth Pages
image
image

### CODE REVIEW

File: src/pages/accountpage/account-page.jsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const AccountPage = () => {
+    return (
+        <>
+            <h1></h1>
```
Comment (coldpigli) [inline src/pages/accountpage/account-page.jsx]: Put in some dummy text for now.

File: src/pages/authpage/signuppage/signup-page.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import { Link } from "react-router-dom"
+
+const Signup = () => {
+    return(
+        <div class="form-main">
```
Comment (coldpigli) [inline src/pages/authpage/signuppage/signup-page.jsx]: change all the "class" attributes to "className"

File: src/pages/authpage/signuppage/signup-page.jsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        <div class="cart-head">Signup</div>
+        <form class="form nav--shadow">
+            <label for="email" class="card-head">
```
Comment (coldpigli) [inline src/pages/authpage/signuppage/signup-page.jsx]: replace 'for' with 'htmlFor'

File: src/pages/authpage/signuppage/signup-page.jsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+            <input class="form-input"
+            type="email" name="" id="email" />
+            <label for="pass" class="card-head">
+                Enter Your Password
+            </label>
+            <input class="form-input" type="password" name="" id="pass" />
+
+            <div class="forgot-pass">
+                <label for="remem" class="remem-fl">
+                    <input type="checkbox" name="" id="remem" />
```
Comment (coldpigli) [inline src/pages/authpage/signuppage/signup-page.jsx]: name is empty here

File: src/pages/authpage/logout/logout.jsx [hunk 5]
```diff
@@ -0,0 +1,14 @@
+const Logout = () => {
+    return (
+        <div class="logout-main">
+            <div class="logout-container">
+                <i class="far fa-check-square green-tick"></i>
```
Comment (coldpigli) [inline src/pages/authpage/logout/logout.jsx]: change class to className

**Review by coldpigli — COMMENTED** [review]
Requested minor changes

**Review by coldpigli — APPROVED** [review]
Looks good now. Go ahead and merge

**Review by rahulcs754 — APPROVED** [review]
The code looks good and clean.

**Pull request merged**